src/ApiHandler.ts

```diff
@@ -61,11 +63,13 @@ export default class ApiHandler {
 			const { method, nonce, signature, signer, isSigned, tip, args } = extrinsic;
 			const hash = u8aToHex(blake2AsU8a(extrinsic.toU8a(), 256));
 
+			const argsWithParsedCalls = parseCalls(args);
```

Comment:
May be best to just move the `parseCalls` function directly to line 72...this will keep the diff "smaller".